blocks/api/raw-handling/test/special-comment-converter.js

```diff
@@ -3,6 +3,18 @@
  */
 import { equal } from 'assert';
 
+function trimWhitespace( string ) {
+	return string.replace( /(  +)|\n|\t/g, '' );
+}
+
+function wrapInP( string ) {
+	return `<p>${ string }</p>`;
+}
+
+function withEmptyP( string ) {
```

Comment:
I'm not sure if this is any create than without this function. `withEmptyP` leases me wondering where exactly that empty paragraph will be.

Reply:
Agreed; removing simplifies things.